Connecting personal archives added (#1826)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeServer/recipients/disconnected-mailboxes/connect-disabled-mailboxes.md

    Exchange will reconnect the disabled mailbox to the corresponding user account.

## Use the Exchange Management Shell to connect a disabled mailbox or personal archive

> Use the **Connect-Mailbox** cmdlet in the Exchange Management Shell to connect a user account to a disabled mailbox. You have to specify the type of mailbox that you're connecting. The following examples show the syntax for reconnecting user, linked, shared, and archive mailboxes.

Reviewer: ```suggestion
Use the **Connect-Mailbox** cmdlet in the Exchange Management Shell to connect a user account to a disabled mailbox. You have to specify the type of mailbox that you're connecting. The following examples show the syntax for reconnecting user, linked, shared, and archived mailboxes.
```

> [!NOTE]
> If you don't include the _Alias_ parameter when you run the **Connect-Mailbox** cmdlet, the value specified in the _User_ or _LinkedMasterAccount_ parameter is used to create the email address alias for the reconnected mailbox.

> This example connects a personal archive to the primary mailbox using the mailbox Guid stored in mailbox database DB01.

Reviewer: ```suggestion
This example connects a personal archive to the primary mailbox using the mailbox GUID stored in mailbox database DB01.
```

```PowerShell
Connect-Mailbox -Identity "95352f8b-e5aa-496f-ac7f-ce93357d7b0c" -Archive -User "Megan Bown" -Database "DB01"
```

If you do not know the name of the personal archive, you can view it in the Exchange Management Shell by running the following command. This example returns all personal archive mailboxes in mailbox database DB01.

```PowerShell
Get-MailboxDatabase "DB01" | Get-MailboxStatistics | Where {($_.DisconnectDate -ne $null) -and ($_.IsArchiveMailbox -eq $true)} | Format-Table DisplayName,MailboxGuid -AutoSize
```

> [!NOTE]

> > You can connect a personal archive mailbox to a different primary mailbox, not matching the previous owner mailbox. Ensure using the _AllowLegacyDNMismatch_ parameter to allow connecting the archive mailbox to a different primary mailbox.

Reviewer: ```suggestion
> You can connect a personal archive mailbox to a different primary mailbox, not matching the previous owner mailbox. Ensure that you are using the _AllowLegacyDNMismatch_ parameter to allow the connection of the archive mailbox to a different primary mailbox.
```
